recipes-azure-iot/azure-identityd/aziot-identityd.inc

```diff
@@ -119,7 +125,14 @@ do_install() {
         -e 's#@libexecdir@#/usr/libexec#g' \
         -e '/Environment=\(.*\)$/d' \
         ${D}${systemd_system_unitdir}/aziot-certd.service
-    install -m 0644     ${S}/cert/aziot-certd/aziot-certd.socket ${D}${systemd_system_unitdir}/aziot-certd.socket
+    install -m 0644     ${S}/cert/aziot-certd/aziot-certd.socket.in ${D}${systemd_system_unitdir}/aziot-certd.socket
+    sed -i -e 's|@user_aziotid@|${USER_AZIOTID}|' \
+	   -e 's|@user_aziotks@|${USER_AZIOTKS}|' \
+	   -e 's|@user_aziotcs@|${USER_AZIOTCS}|' \
+	   -e 's|@user_aziottpm@|${USER_AZIOTTPM}|' \
+	   -e 's|@socket_dir@|${SOCKET_DIR}|' \
+	${D}${systemd_system_unitdir}/aziot-certd.service ${D}${systemd_system_unitdir}/aziot-certd.socket
```

Comment:
seems we should do that as a function ... because you do the same sed step for all socket files

Reply:
right, and not only for socket files but also for service files.
done